components/brave_wallet_ui/components/desktop/sweepstakes-banner/style.ts

```diff
@@ -1,5 +1,10 @@
 import styled from 'styled-components'
-export { CloseButton } from '../popup-modals/style'
+import { CloseButton as OriginalCloseButton } from '../popup-modals/style'
+
+export const CloseButton = styled(OriginalCloseButton)`
+    color: #212529;
+    background-color: #212529;
```

Comment:
Is there a theme variable we can use here?

Reply:
+1

Reply:
I was using the theme variables until i realized that the color is the same between light and dark modes